### ISSUE TITLE
Can't set shortcut in XCode

### ISSUE THREAD

**Issue by tachoknight** [nicklockwood/SwiftFormat#608]
Aloha-

I installed `swiftformat-for-xcode` from Homebrew and, while everything works, I can't seem to set a keyboard shortcut for it; I set one in Xcode preferences but it doesn't seem to take in that the shortcut doesn't appear next to the menu item, and pressing the shortcut key doesn't do anything. I would assume this was an Xcode issue but I can set shortcuts for other things properly, I checked that the "SwiftFormat for Xcode" Extension is installed and "Xcode Source Editor" is checked, tried restarting Xcode a bunch of times, etc. Might there be a trick that I'm missing?

Fantastic software, it's really useful for keeping everything nice and formatted; major thanks and kudos for it!

**Comment by nicklockwood** [comment]
I've not encountered this specific issue - it seems to work correctly for me.

Previously, problems with SwiftFormat not showing up in the menu have been fixed by moving Xcode.app into a different folder and then moving it back again - might be worth a try?

**Comment by mkoorn** [comment]
I had the same issue. swiftformat had been working as expected for some time.
After installing using brew (deleted the previous version) the shortcut stopped working.

Only after complete uninstall using clean my mac and downloading the latest version via releases, did the shortcut work again.

**Comment by tachoknight** [comment]
So, I "fixed it", I removed the brew versions, I tried using the release builds, I even tried rebuilding it myself, all to no avail. What I noticed was that at some point additional choices was added, there is "Format File" and "Format Entire File", as well as "Format Selection" and "Format Selected Source". 

I tested just on the file option, so I tried "Format Entire File", but no luck. I switched the shortcut to "Format File" and tada, it's there and works. I _think_ one problem I was having is that I was not removing the keybindings from Xcode before doing all the work above.
